sympy/stats/tests/test_rv.py

```diff
@@ -210,16 +211,27 @@ def test_Sample():
     assert sample(X) in [1, 2, 3, 4, 5, 6]
     assert isinstance(sample(X + Y), float)
 
-    assert P(X + Y > 0, Y < 0, numsamples=10).is_number
+    # XXX: This should be fixed in lambdify to not use numpy functions from the
+    # scipy module.
+    #
+    # DeprecationWarning: scipy.greater is deprecated and will be removed
+    # in SciPy 2.0.0, use numpy.greater instead
+    # DeprecationWarning: scipy.sin is deprecated and will be removed in SciPy
+    # 2.0.0, use numpy.sin instead
+    with ignore_warnings(DeprecationWarning):
+        assert P(X + Y > 0, Y < 0, numsamples=10).is_number
```

Comment:
This is a significant problem in `lambdify`. The scipy module has lots of functions that are really just imported from numpy like `scipy.sin` etc. These are all now deprecated. That means that `lambdify` should always prefer a function like `numpy.sin` over `scipy.sin` even if the module is set to "scipy".

I'm not sure how easy it is to fix this in lambdify because it isn't clear to me exactly how these names are picked from one module or the other e.g.:
```python
In [8]: f = lambdify(x, sin(x), 'scipy')

In [9]: print(inspect.getsource(f))
def _lambdifygenerated(x):
    return sin(x)
```